Packs/Druva/Integrations/Druva/Druva.py

```diff
@@ -339,9 +429,16 @@ def main():
             restore_id = demisto.args().get('restore_id')
             return_outputs(*Druva_Restore_Status(clientObj, restore_id))
 
-        if command == 'druva-endpoint-decommission':
-            resource_id = demisto.args().get('resource_id')
-            return_outputs(*Druva_Decommission(clientObj, resource_id))
+
+
+        if command == 'hi-command':
```

Comment:
Remove. Also, remove all other commented out code you have in the file, for example:

```python
# return_outputs(*Druva_ListQuarantine_Snapshots_Command(clientObj, resource_id, range_id))
```

Reply:
Commented code is removed